tasks/main.yml

```diff
@@ -54,6 +54,13 @@
         enabled: yes
   when: not __postgresql_conf.stat.exists
 
+# start already installed and initialized server
+- name: Enable and start existing instance of postgresql server
+  service:
+    name: postgresql
+    state: started
+    enabled: yes
```

Comment:
This should typically be done in a handler, and only when a new instance of postgresql was added or modified in some way (and then use `state: restarted` instead of `started`) - can you explain what you're trying to do here?

Reply:
It should handle situations when the user has already installed and initialized PostgreSQL but it is stopped. So it needs to be started before any further actions.